mozilla/fx-private-relay
# Add Fluent library to the website project

## Summary

This PR adds Fluent (via [django-ftl](https://github.com/django-ftl/django-ftl) to the Django app

TODO: 

- [x] Footer
- [x] FAQ
- [x] Profile (logged in)
- [x] Survey Banner
- [x] ~Add Bento box strings [See Monitor bento.ftl implementation](https://github.com/mozilla/blurts-server/blob/main/locales/en/bento.ftl)~ Filed a follow-up issue here:  #822
- [x] ~Update hero images (Text is embedded)~ The strings are added to `app.ftl`, but we'll need to bust up the art to make this work fully. Filed a follow-up issue here:  #821

## Testing 

Verify English:
- Set browser language as a `en*` variant. 
- Navigate to homepage: http://localhost:8000/
- Expected: Headline should have `[EN]` string visible (See screenshot)



Verify German (non-English): 
_Note - To verify German, please checkout this commit: 2cc98dc0efac18306bb8f22e5cb6ae82f71f9f8d_
- Set browser language as a `de*` variant. 
- Navigate to homepage: http://localhost:8000/
- Expected: Headline should have `[DE]` string visible

## Screenshots

<img width="706" alt="image" src="https://user-images.githubusercontent.com/2692333/115604952-918b5d80-a2a7-11eb-86c8-eb21816e3017.png">
